developer-docs-site/docs/tutorials/first-move-module.md

```diff
@@ -9,13 +9,13 @@ import TabItem from '@theme/TabItem';
 
 # Your first Move Module
 
-This tutorial details how to write, compile, test, publish, and interact with Move Modules on the Aptos Blockchain. The steps to doing so:
+This tutorial details how to write, compile, test, publish and interact with Move Modules on the Aptos Blockchain. The steps to doing this are:
 
-* Write, compile, and test the Move Module
-* Publish the Move Module to the Aptos Blockchain
-* Initialize and interact with resources of the Move Module
+1. Write, compile and test the Move Module
```

Comment:
no, I am sorry, I cannot accept this, sincerely. the oxford comma is mandatory.

Reply:
Incredible! 😄